control/statesp.py

```diff
@@ -1331,3 +1331,105 @@ def ssdata(sys):
     """
     ss = _convertToStateSpace(sys)
     return ss.A, ss.B, ss.C, ss.D
+
+
+def consistent_dimensions(A=None, B=None, C=None, D=None):
+    """
+    Return a boolean stating if dimensions of the state-space matrices are consistent.
+
+    A state space representation should verify:
+    - A square matrix [n_states x n_states]
+    - B [n_states x n_inputs]
+    - C [n_outputs x n_states]
+    - D [n_outputs x n_inputs]
+    Parameters
+    ----------
+    A: ndarray - optional
```

Comment:
Per [numpydocs](http://numpydoc.readthedocs.io), this should be "A : ndarray, optional".  Same below.